### DISCUSSION TITLE
Light Dismiss

### DISCUSSION THREAD

**Discussion by rinaok** [floating-ui/floating-ui#1857] [Q&A]
Hey :)

Is there a future plan to support `light dismiss`?

Thanks!

**Comment by FezVrasta** [comment]
Could you add some more details to your request please?

**Reply by yinonov** [reply]
https://open-ui.org/components/popup.proposal-v1#light-dismissal explains the concept. it is basically just clicking outside the popup closes it

**Reply by atomiks** [reply]
What needs to be clarified is what package is @rinaok using? The vanilla positioning package does not offer any "interactions", it only offers positioning. So light dismiss needs to be implemented yourself. The React Interactions package has a hook for it though.

**Reply by yinonov** [reply]
it's the vanilla positioning package :) thanks!
